lib/commanded/event_store/event_data.ex

```diff
@@ -7,8 +7,8 @@ defmodule Commanded.EventStore.EventData do
   @type uuid :: String.t()
 
   @type t :: %Commanded.EventStore.EventData{
-          causation_id: uuid(),
-          correlation_id: uuid(),
+          causation_id: uuid() | nil,
+          correlation_id: uuid() | nil,
```

Comment:
```suggestion
          correlation_id: uuid(),
```
FYI, it looks like `correlation_id` will be automatically generated if not specified. See [here](https://github.com/commanded/commanded/blob/master/lib/commanded/commands/router.ex#L515).